subworkflows/local/binning.nf

```diff
@@ -36,17 +25,16 @@ workflow BINNING {
     // generate coverage depths for each contig
     ch_summarizedepth_input = assemblies
                                 .map { meta, assembly, bams, bais ->
-                                        def meta_new = meta.clone()
+                                        def meta_keys = meta.keySet()
+                                        def meta_new = meta + meta.subMap(meta_keys)
```

Comment:
Wouldn't `meta_new` be the same as `meta`? 🤔

Reply:
Ehhh, yes :sweat_smile: